hadoop-hdds/interface-server/src/main/proto/ScmServerSecurityProtocol.proto

```diff
@@ -195,6 +200,15 @@ message SCMListCertificateResponseProto {
     repeated string certificates = 2;
 }
 
+message SCMGetAllRootCaCertificatesResponseProto {
+    enum ResponseCode {
```

Comment:
In the general SCMSecurityRequest/Response where we have the different request/Response types, and where you have added this protocol message as optional on the 11th position in line 86, we have a couple of fields: success, message, Status. I believe we should use that to convey success or failure related information, the submitRequest method on the client side, and the request handling on the server has the facilities to deal with errors and succes via these. Can you please use those instead of a specific ResponseCode here?

Reply:
I'm not sure if I managed to resolve this successfully. Do I understand it correctly that these specific response codes should only be used when we want to specify something that is not already covered in SCMSecurityRequest/Response?

Now the command type is set according to the rest of the examples and the ResponseCode handling should be dealt with in the central location.